tools/cli/help.txt

```diff
@@ -171,6 +172,13 @@ versions. It will not update the version of meteor. To update individual
 packages (for example: 'foo:awesome') pass in their names instead, with no
 options. ('meteor update foo:awesome').
 
+Passing --all-packages will update all packages, including indirect
+dependencies, to their latest compatible versions. This is similar to passing
+the names of all project packages to the meteor update command. Note that
+updated packages must meet the constraints imposed by other packages used by
+the project. This means that some packages, while updated, might still not be
+updated to their most recent version.
+
```

Comment:
Thoughts on adding one more thing here?:

```
<new paragraph>
To find more information as to why a package is kept from upgrading, run:

    meteor add package@<target-version>

```

... or whatever similar will fit in the 80 character mark. :wink: